plugins/webhooks/index.js

```diff
@@ -310,6 +311,93 @@ function pullRequestEvent(pluginOptions, request, reply, parsed) {
         .catch(err => reply(boom.wrap(err)));
 }
 
+/**
+ * Check the pipeline have triggered job or not
```

Comment:
Check if the pipeline has a triggered job or not

Reply:
Thank you. I will fix